pkg/apis/work/v1alpha2/binding_types.go

```diff
@@ -142,6 +158,47 @@ type TargetCluster struct {
 	Replicas int32 `json:"replicas,omitempty"`
 }
 
+// GracefulEvictionTask represents an graceful eviction task.
+type GracefulEvictionTask struct {
+	// Name of the cluster where the eviction perform from.
+	Name string `json:"name"`
+
+	// Reason contains a programmatic identifier indicating the reason for the eviction.
+	// Producers may define expected values and meanings for this field,
+	// and whether the values are considered a guaranteed API.
+	// The value should be a CamelCase string.
+	// This field may not be empty.
+	// +required
+	// +kubebuilder:validation:MaxLength=32
+	// +kubebuilder:validation:MinLength=1
+	// +kubebuilder:validation:Pattern=`^[A-Za-z]([A-Za-z0-9_,:]*[A-Za-z0-9_])?$`
+	Reason string `json:"reason" protobuf:"bytes,5,opt,name=reason"`
+
+	// Message is a human-readable message indicating details about the eviction.
+	// This may be an empty string.
+	// +optional
+	// +kubebuilder:validation:MaxLength=1024
+	Message string `json:"message,omitempty"`
+
+	// Producers indicates the controller who triggered the eviction.
+	// +required
+	Producers string `json:"producers"`
```

Comment:
1. Should it be `Producer` or `Producers` here? If it's `Producers`, you'd better define delimiter for spliting each producer. And clarifing the behavior of "mutiple producers generating the same `GracefulEvictionTask` would be very helpful."
2. To avoid hard coding strings for producers in multiple locations, you may need to enumerate valid producers, or at least find a unified place to define consts.

Reply:
Done. Added a const in new push.